Template documentation emphasises container usage (#539)
nf-core/tools: CHANGELOG.md

* Removed the requirement for R in the conda environment
* Make `params.multiqc_config` give an _additional_ MultiQC config file instead of replacing the one that ships with the pipeline
* Ignore only `tests/` and `testing/` directories in `.gitignore` to avoid ignoring `test.config` configuration file

> * Rephrase quickstart and usage templates to promote usage of containers over conda for more stable reproducibility

Reviewer: ```suggestion
* Rephrase docs to promote usage of containers over Conda to ensure reproducibility
```
